gui/web/src/Form.js

```diff
@@ -0,0 +1,321 @@
+import React, { Component } from 'react';
+import styles from './components/templates/Form/Form.module.scss';
+import grid from './components/_settings/grid.module.scss';
+import Input from './components/atoms/Input/Input';
+import Label from './components/atoms/Label/Label';
+import SectionTitle from './components/atoms/SectionTitle/SectionTitle';
+import Switch from './components/atoms/Switch/Switch';
+import SegmentedControl from './components/atoms/SegmentedControl/SegmentedControl';
+import SectionDescription from './components/atoms/SectionDescription/SectionDescription';
+import Button from './components/atoms/Button/Button';
+import Select from './components/atoms/Select/Select';
+import FieldItem from './components/molecules/FieldItem/FieldItem';
+import ScreenHeader from './components/molecules/ScreenHeader/ScreenHeader';
+import AdvancedWrapper from './components/molecules/AdvacedWrapper/AdvancedWrapper';
+import FormSection from './components/molecules/FormSection/FormSection';
+import FieldGroup from './components/molecules/FieldGroup/FieldGroup';
+import PriceFeedSelector from './components/molecules/PriceFeedSelector/PriceFeedSelector';
+import PriceFeedTitle from './components/molecules/PriceFeedTitle/PriceFeedTitle';
+import PriceFeedFormula from './components/molecules/PriceFeedFormula/PriceFeedFormula';
+import RepeaterField from './components/molecules/RepeaterField/RepeaterField';
+
+class Form extends Component {
+  render() {
+    return (
+      <div>
+        <div className={grid.container}>
```

Comment:
we may need a way to make this form smaller, maybe autogenerated from the Kelp backend based on configs, but ok for now. let's see how much repetition we end up having.